client/tests/acceptance/user-can-click-landuse-form-edit-test.js

```diff
@@ -0,0 +1,132 @@
+  setupMirage(hooks);
+
+  hooks.beforeEach(() => {
+    authenticateSession({
+      emailaddress1: 'me@me.com',
+    });
+  });
+
+  test('User can add an applicant on the landuse form', async function(assert) {
+    this.server.create('project', 1, {
+      packages: [this.server.create('package', 'toDo', 'landuseForm')],
+    });
+
+    await visit('/landuse-form/1/edit');
+
+    await click('[data-test-add-applicant-button]');
+    await fillIn('[data-test-input="dcpFirstname"]', 'Tess');
+    await fillIn('[data-test-input="dcpLastname"]', 'Ter');
+    await fillIn('[data-test-input="dcpEmail"]', 'tesster@planning.nyc.gov');
+    await click('[data-test-save-button]');
+
+    assert.equal(this.server.db.applicants.firstObject.dcpFirstname, 'Tess');
+
+    assert.equal(currentURL(), '/landuse-form/1/edit');
+  });
+
+  test('User can add and a related action on the landuse form', async function(assert) {
+    this.server.create('project', 1, {
+      packages: [this.server.create('package', 'toDo', 'landuseForm')],
+    });
+
+    await visit('/landuse-form/1/edit');
+
+    // fill out other necessary fields for saving
+    await click('[data-test-add-applicant-button]');
+    await fillIn('[data-test-input="dcpFirstname"]', 'Tess');
+    await fillIn('[data-test-input="dcpLastname"]', 'Ter');
+    await fillIn('[data-test-input="dcpEmail"]', 'tesster@planning.nyc.gov');
+
+    // add and fill out fields for related action
+    await click('[data-test-add-related-action-button]');
+    await click('[data-test-action-completed="true"]');
+    await fillIn('[data-test-input="dcpReferenceapplicationno"]', '12345678');
+    await fillIn('[data-test-input="dcpApplicationdescription"]', 'applicant description');
+    await fillIn('[data-test-input="dcpDispositionorstatus"]', 'disposition or status');
+    await fillIn('[data-test-input="dcpCalendarnumbercalendarnumber"]', 'calendar number');
+    await fillIn('[data-test-input="dcpApplicationdate"]', 'application date');
+    await click('[data-test-save-button]');
+
+    assert.equal(this.server.db.applicants.firstObject.dcpFirstname, 'Tess');
+    assert.equal(this.server.db.relatedActions.firstObject.dcpReferenceapplicationno, '12345678');
+  });
+
+  test('user can remove applicants on landuse form', async function(assert) {
+    const project = this.server.create('project', 1, {
+      packages: [this.server.create('package', 'toDo', 'landuseForm')],
+    });
+    const { landuseForm } = project.packages.models[0];
+    // create an applicant model
+    const serverSideApplicant = this.server.create('applicant', 'organizationApplicant', { landuseForm });
+    // get the reference to the model instance
+    const applicant = await this.owner.lookup('service:store').findRecord('applicant', serverSideApplicant.id);
+
+    await visit('/landuse-form/1/edit');
+
+    await assert.equal(applicant.hasDirtyAttributes, false);
+    await assert.equal(applicant.isDeleted, false);
```

Comment:
1. No need to await these assertions — they are synchronous and don't return any promises.
2. Why do we need to assert this model state? Is there some reason we can't trust that the ember-data API might not work?

Reply:
These tests were copied over from how we were doing them in the pas-form, might be outdated code. I don't have any problem removing them

Reply:
Oh okay! Yeah, just isn't necessary. Up to you.